chain-tx-enclave-next/mls/Cargo.toml

```diff
@@ -19,6 +19,7 @@ aead = "0.2.0"
 rand = "0.7"
 ra-client = { path = "../enclave-ra/ra-client" }
 subtle = "2.2.3"
+rand_core = "0.5"
```

Comment:
why rand_core? isn't that exposed from rand?

Reply:
removed